lib/src/sfu_connection.dart

```diff
@@ -30,6 +30,9 @@ class SfuConnection {
   late final RTCPeerConnection _peerConnection;
   late final RTCRtpTransceiver _audio;
   late final RTCRtpTransceiver _video;
+  RTCPeerConnectionState? _connectionState;
+
+  bool get isConnectionFailed => _connectionState == RTCPeerConnectionState.RTCPeerConnectionStateFailed;
```

Comment:
These are not used yet, but I can see myself want to know which SFU connection has failed.